docs/partners/adopt.md

```diff
@@ -28,19 +28,17 @@ The process to Adopt-a-Pack takes under 5 minutes! Follow the step by step instr
 
 <br/>
 
-Please follow the below steps if you are working on a cloned Github repository from an IDE: 
+Please follow the below steps if you are working on a cloned GitHub repository from an IDE: 
 
 1. Locate your company's pack folder and open the `README.md` file. Paste the below text into the file: 
 
     ```
-    Note: Support for this pack moved to the partner on MONTH, DAY, YEAR.
-    
-    Please contact the partner directly via the support link on the right.
+    Note: Support for this Pack will be moved to the Partner on MONTH, DAY, YEAR.
     ```
     Make sure you change the `MONTH`, `DAY`, and `YEAR` to the appropriate date that is 90 days from your submittal date.
 
     - Once complete, save these changes and run `demisto-sdk update-release-notes -i <path to pack> -f` to update the release notes. See [documentation](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md). After the command has been completed, it will create the new release note Markdown file in the `ReleaseNotes` folder and update the version number in `pack_metadata.json`. Before continuing, you need to add the following text to the release note: `Start of adoption process, update to readme file`
```

Comment:
```suggestion
    - Once complete, save these changes and run `demisto-sdk update-release-notes -i <path to pack> -f` to update the release notes. See [documentation](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md). After the command has been completed, it will create the new release note Markdown file in the `ReleaseNotes` folder and update the version number in `pack_metadata.json`. Before continuing, you need to add the following text to the release note: `Start of adoption process, visit the pack readme for more info`
```

Reply:
Let's also fix this ☝️  in the Github UI flow